modules/http4s-kernel/src/smithy4s/http4s/kernel/MessageEncoder.scala

```diff
@@ -73,6 +73,11 @@ object MessageEncoder {
       val headers = toHeaders(metadata.headers)
       val status = metadata.statusCode
         .flatMap(Status.fromInt(_).toOption)
+        .filter(_.responseClass match {
```

Comment:
There is `isSuccess` on status: https://www.javadoc.io/doc/org.http4s/http4s-docs_2.13/latest/org/http4s/Status.html#isSuccess:Boolean

which is implemented in this way:
```
case object Informational extends ResponseClass { val isSuccess = true }
  case object Successful extends ResponseClass { val isSuccess = true }
  case object Redirection extends ResponseClass { val isSuccess = true }
  case object ClientError extends ResponseClass { val isSuccess = false }
  case object ServerError extends ResponseClass { val isSuccess = false }
  ```
  
  I don't know if you meant to exclude `Informational` or if you did not know about `isSuccess`. But if it's the former, I'd add a comment
  
  if it's the latter, I'd use is success

Reply:
I intended to exclude `Informational` based on the comment https://github.com/disneystreaming/smithy4s/issues/916#issuecomment-1516512415

Reply:
I agree with @daddykotex in that we should be using `isSuccess`

Reply:
Thanks for feedbacks. I replaced it with `isSuccess`.